guides/common/modules/proc_managing-image-mode-hosts-and-booted-container-images.adoc

```diff
@@ -0,0 +1,23 @@
+[id="Managing_image_mode_hosts_and_booted_container_images_{context}"]
+= Managing image mode hosts and booted container images
+
+These actions streamline the management of image mode hosts, providing flexibility and control over host operations through {Project}.
+
+.Procedure
+. Navigate to *Content* > *Booted Container Images* to view all images used by image mode hosts.
+This page displays the spread of digests under specific image paths.
+More digests under an image path indicate greater drift in the host ecosystem, ideally minimized by ensuring all hosts run the most up-to-date image version.
+. Click the host count to navigate to the *All Hosts Listing* page, which identifies the hosts associated with each specific image and displays the number of hosts using each digest.
```

Comment:
Note: this is only on the _new_ hosts overview page, which is not the default yet.